Parcare/Services/ParkingLotService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Parking.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Parking.Services
+{
+    public class ParkingLotService
+    {
+        private List<ParkingLotModel> _parkedCars;
+        private int _maxSlots;
+        private int _availableSlots;
+
+        public ParkingLotService(int totalSlots)
+        {
+            this._parkedCars = new List<ParkingLotModel>();
+            this._maxSlots = totalSlots;
+            this._availableSlots = _maxSlots - _parkedCars.Count;
+        }
+
+        public bool IsParkingPossible()
+        {
+            return _availableSlots > 0;
```

Comment:
either have the logic here, not in constructor because you have to recalculate each time you park a car the available slots, or you use a _isParkingPossible instead of available slots